<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1302</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interpolacion Lineal de Alto Orden by Edward Angelino</string>
  </property>
  <widget class="QTableView" name="dxuncView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>621</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="dxattView">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>50</y>
     <width>621</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tofutView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>621</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="toattView">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>370</y>
     <width>621</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nomdxunc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>561</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nomtofut">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>551</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>340</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>660</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Calcular</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nomdxatt">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>561</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_4">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>20</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nomtoatt">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>340</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>toatt.txt</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>340</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Resultados :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DXUNC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>0</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DXATT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TOFUT</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_5">
   <property name="geometry">
    <rect>
     <x>1190</x>
     <y>660</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Graficar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
